lib/shared/config/cli-flags.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var yargs = require('yargs');
+var camelCase = require('lodash.camelcase');
+
+function mergeToCliFlags(cliFlags, config, cliOptions) {
+  var argv = yargs(process.argv.slice(2)).argv;
+  var cfgFlags = config.flags || {};
+
+  var optNames = ['silent'];
+  optNames.filter(excludeArgsByUser).map(camelCase).forEach(copyConfig);
+
+  function excludeArgsByUser(name) {
```

Comment:
This function checks if `name` is not included in command-line arguments. 

If all flags specified by config files are reflected to `opts`, flags by command-line args are overridden. I think that specifying by command-line args should be given preference over specifying by config files.